sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -107,6 +113,11 @@ AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
     }
   };
 
+  if (!az_http_policy_retry_options_validate(&self->_internal.options.retry))
```

Comment:
We don't need this

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/413#discussion_r387288780

Reply:
Done :)